src/cmd/sv-rollout/svrestarter.go

```diff
@@ -102,7 +107,7 @@ func (s *SvRestarter) notifyResult(result error) {
 	case ErrRestartPreempted:
 		s.log("was not required to restart in time", true)
 	default:
-		panic(result)
+		s.log(fmt.Sprintf("Unexpected error handled, likely a bug: %s : %s", reflect.TypeOf(result).String(), result), true)
```

Comment:
I agree, but a bug in preempt shouldn't cause failures (per the design of it). It should be logged and monitored as an exception. But I dont feel strongly about this so i'll just add `panic` back